pipeline/merger/src/main/scala/weco/pipeline/merger/rules/TargetPrecedence.scala

```diff
@@ -2,18 +2,36 @@ package weco.pipeline.merger.rules
 
 import weco.catalogue.internal_model.work.Work
 import weco.catalogue.internal_model.work.WorkState.Identified
-
-object TargetPrecedence {
-  import WorkPredicates._
-
-  // This is the canonical list of the order in which we try to select target works
-  private val targetPrecedence = Seq(
+import weco.pipeline.merger.rules.WorkPredicates.{
+  physicalSierra,
+  sierraElectronicVideo,
+  sierraWork,
+  singlePhysicalItemCalmWork,
+  teiWork,
+  WorkPredicate
+}
+object TeiTargetPrecedence extends BaseTargetPrecedence {
+  override val targetPrecedence: Seq[WorkPredicate] = Seq(
     teiWork,
     singlePhysicalItemCalmWork,
     sierraElectronicVideo,
     physicalSierra,
     sierraWork
   )
+}
+object DefaultTargetPrecedence extends BaseTargetPrecedence {
+  override val targetPrecedence: Seq[WorkPredicate] = Seq(
+    singlePhysicalItemCalmWork,
+    sierraElectronicVideo,
+    physicalSierra,
+    sierraWork
+  )
+}
+trait BaseTargetPrecedence {
+  import WorkPredicates._
+
+  // This is the canonical list of the order in which we try to select target works
+  val targetPrecedence: Seq[WorkPredicate]
```

Comment:
Minor: the order of definitions in this file seems backwards.

I'd expect it to go:

* Base trait
* Default implementation
* Specialised implementations